apps/crc_interactive.py

```diff
@@ -97,6 +98,11 @@ def _validate_arguments(self, args: Namespace) -> None:
         if args.mpi and (not args.partition == 'compbio') and args.num_nodes < self.min_mpi_nodes:
```

Comment:
The `compbio` partition doesn't exist anymore. This is a good opportunity to update the conditional.